docs/docs/responses.mdx

```diff
@@ -167,8 +167,10 @@ Each button in the list of `buttons` should have two keys:
 :::note bypass nlu with buttons
 It's common to use buttons as a shortcut to bypass the machine-learning-based NLU interpreter.
 Messages starting with `/` are sent straight to the
-`RegexInterpreter`, which expects NLU input in a shortened `/intent{entities}` format.
-In the example above, if the user clicks a button, the user input
+`RegexInterpreter`, which expects NLU input in a shortened `/intent{entities}` format, e.g.
+`payload: "/inform{{\"planet\": \"saturn\"}}"`.
```

Comment:
might look better as a code block? and do we really need the double brackets?

Reply:
I can make it a code block. Yes, this fails without the double brackets, which is what triggered me to write this.